From 25168ce6bd05cdfa2aef34ebc5b967465cf94d15 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 24 Dec 2020 15:27:16 +0100
Subject: [PATCH] vim: add libiconv dependency

While compiling vim-full, it fails because of this error:
msgfmt: Cannot convert from "ISO-8859-1" to "UTF-8". msgfmt relies on iconv(). This version was built without iconv().

This fixes: https://github.com/openwrt/packages/issues/14328

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 utils/vim/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/vim/Makefile b/utils/vim/Makefile
index adc3751a7..947db44a4 100644
--- a/utils/vim/Makefile
+++ b/utils/vim/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vim
 PKG_VERSION:=8.2
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 VIMVER:=82
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
@@ -31,7 +31,7 @@ include $(INCLUDE_DIR)/host-build.mk
 define Package/vim/Default
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses +libiconv
   TITLE:=Vi IMproved - enhanced vi editor
   URL:=http://www.vim.org/
   SUBMENU:=Editors
-- 
2.25.1

